### ISSUE TITLE
Playwright Java - is there a way to use system node version ?

### ISSUE THREAD

**Issue by mEklavya** [microsoft/playwright#16038]
Playwright.create() is failing as it is installing the node.exe in the temp directory which the user is not allowed to run. Is there a way not to use this temp path and use system node js installed in program files

![image](https://user-images.githubusercontent.com/42370888/181646341-b7fa9419-e853-491d-9dca-d13843007868.png)

**Comment by yury-s** [comment]
No, the node.js and the driver files is essential part of Playwright. You can configure alternative location of the temp directory by providing `playwright.driver.tmpdir` java system property (see https://github.com/microsoft/playwright-java/issues/728 for more details). Would that work for you?
